### PR TITLE
Update porting content

### PR DESCRIPTION
**Pull request by ronak2ram**
## This PR is a:

- [ ] New topic
- [x] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will show updated content to magento contributors. 

## Additional information

List all affected URLs 

- https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#porting
- https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#porting
- https://devdocs.magento.com/guides/v2.1/contributor-guide/contributing.html#porting

whatsnew
Update [community contribution guidelines](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#porting) about porting

### CODE REVIEW

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/contributor-guide/contributing.md [hunk 1]
```diff
@@ -226,7 +226,7 @@ We provide two options to create back-ports and up-ports for your code contribut
 - [Manual porting](#porting-manual) - Manual process requiring a strong understanding of Git
 
 {: .bs-callout .bs-callout-info }
-Creating back-ports and up-ports are recommended and a best practice, but not required to contribute code. Anyone can create a back-port and up-port for an already merged pull request.
+Creating back-ports and up-ports are recommended and a best practice, but not required to contribute code. Anyone can create a back-port and up-port for an already merged pull request but for the duplicate ports we give a high priority to merged pull request author.
```
Comment (jeff-matthews) [inline guides/v2.1/contributor-guide/contributing.md]:
```suggestion
Creating back-ports and up-ports are recommended and a best practice, but not required to contribute code. Anyone can create a back-port and up-port for a merged pull request, however, original pull request authors receive higher priority if there are duplicate ports.
```

File: guides/v2.2/contributor-guide/contributing.md [hunk 2]
```diff
@@ -228,7 +228,7 @@ We provide two options to create back-ports and up-ports for your code contribut
 - [Manual porting](#porting-manual) - Manual process requiring a strong understanding of Git
 
 {: .bs-callout .bs-callout-info }
-Creating back-ports and up-ports are recommended and a best practice, but not required to contribute code. Anyone can create a back-port and up-port for an already merged pull request.
+Creating back-ports and up-ports are recommended and a best practice, but not required to contribute code. Anyone can create a back-port and up-port for an already merged pull request but for the duplicate ports we give a high priority to merged pull request author.
```
Comment (jeff-matthews) [inline guides/v2.2/contributor-guide/contributing.md]:
```suggestion
Creating back-ports and up-ports are recommended and a best practice, but not required to contribute code. Anyone can create a back-port and up-port for a merged pull request, however, original pull request authors receive higher priority if there are duplicate ports.
```

File: guides/v2.3/contributor-guide/contributing.md [hunk 3]
```diff
@@ -227,7 +227,7 @@ We provide two options to create back-ports and up-ports for your code contribut
 - [Manual porting](#porting-manual) - Manual process requiring a strong understanding of git
 
 {: .bs-callout .bs-callout-info }
-Creating back-ports and up-ports are recommended and a best practice, but not required to contribute code. Anyone can create a back-port and up-port for an already merged pull request.
+Creating back-ports and up-ports are recommended and a best practice, but not required to contribute code. Anyone can create a back-port and up-port for an already merged pull request but for the duplicate ports we give a high priority to merged pull request author.
```
Comment (jeff-matthews) [inline guides/v2.3/contributor-guide/contributing.md]:
```suggestion
Creating back-ports and up-ports are recommended and a best practice, but not required to contribute code. Anyone can create a back-port and up-port for a merged pull request, however, pull request authors receive higher priority if there are duplicate ports.
```

File: guides/v2.1/contributor-guide/contributing.md (hunk 1 above)
Comment (ronak2ram) [reply guides/v2.1/contributor-guide/contributing.md]: Hello @jeff-matthews 
Content updated

File: guides/v2.2/contributor-guide/contributing.md (hunk 2 above)
Comment (ronak2ram) [reply guides/v2.2/contributor-guide/contributing.md]: Hello @jeff-matthews 
Content updated

File: guides/v2.3/contributor-guide/contributing.md (hunk 3 above)
Comment (ronak2ram) [reply guides/v2.3/contributor-guide/contributing.md]: Hello @jeff-matthews 
Content updated

**Comment by jeff-matthews** [comment]
Thanks @ronak2ram!

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**